1.0/profile.ttl

```diff
@@ -0,0 +1,63 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    dcterms:identifier "geosparql" ;
+    dcterms:created "2009-10-27"^^xsd:date ;
+    dcterms:modified "2012-07-19"^^xsd:date ;
+    dcterms:issued "2020-xx-xx"^^xsd:date ;
+    dcterms:publisher <https://www.ogc.org> ;
+    dcterms:title "GeoSPARQL Specification"@en ;
+    dcterms:description "This profile supports representing and querying geospatial data on the Semantic Web. GeoSPARQL defines a vocabulary for representing geospatial data in RDF, and it defines an extension to the SPARQL query language for processing geospatial data. It is presented in a series of artefacts: the vocabulary as an RDF ontology and the specification as a PDF document." ;
```

Comment:
"This profile supports representing and querying geospatial data on the Semantic Web": I would say GeoSPARQL does that, and the profile describes a version of GeoSPARQL as a set of resources.

Reply:
Absolutely correct: a copy and paste error. I have updated the description and it now matches that of the 1.1 profile too which was formulated along the lines you suggest. See Commit https://github.com/opengeospatial/ogc-geosparql/pull/58/commits/b05e95fb7d392cee3519ad5389416d4d7f0bfd9a